### PR TITLE
`unit-case` rule

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: src/rules/unit-case/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# unit-case
+
+Specify lowercase or uppercase for units.
+
+```css
+    a { width: 10px; }
+/**              ↑
+ *          These units */
```
Comment (jeddy3) [inline src/rules/unit-case/README.md]: A minor nit-pick, but can you align the "s" of "units" with the arrow please.

**Comment by jeddy3** [comment]
You're getting very very good at writing these rules @evilebottnawi ! :) I can spot only one minor request.

@davidtheclark Can you spot anything else?

**Comment by evilebottnawi** [comment]
@jeddy3 in this rule or other rules? WIP now on `property-case`.

**Comment by jeddy3** [comment]
> @jeddy3 in this rule or other rules?

I'm not sure what you mean. My only request is to align the comment in the docs example.

> WIP now on property-case.

Excellent! :)

**Comment by davidtheclark** [comment]
This looks great, thanks!

**Pull request merged**